packages/lit-html/src/static.ts

```diff
@@ -131,7 +131,10 @@ export const withStatic =
         s += staticValue + strings[++i];
         hasStatics = true;
       }
-      dynamicValues.push(dynamicValue);
+      // If the last value is static, we don't need to push it.
+      if (i !== l) {
+        dynamicValues.push(dynamicValue);
```

Comment:
Looking back at the original code, I don't love that `dynamicValue` apparently can hold a static value. Nice fix for now, but makes me want to clean this up later.